### ISSUE TITLE
Click on Hour or Minute button triggers form action & some styling

### ISSUE THREAD

**Issue by KoljaL** [mskocik/svelty-picker#108]
Great component, thanks a lot :-)

Is it possible to add a preventDefault in here: `() => $$invalidate(5, isMinuteView = false)`

Styling:   
I use a global CSS and then the with of the table is a little to much and the tick-buttons getting a margin/padding.
So I had to reset it:

```css
.std-calendar-wrap {
	width: max-content !important;
}
button.sdt-tick {
	margin: 0;
	padding: 0;
}
```

**Comment by mskocik** [comment]
Easy to fix, but I am currently working on v4, so... later this week probably.

Tick buttons have no padding/margin defined. They just have defined width and height which are needed, because they are absolutely positioned. But if the CSS updates you did work for you, then great!

**Comment by mskocik** [comment]
Form submit issue fixed in both `v4` and `3.6.1`
